RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This release makes our docstring style more consistent, thanks to
+:pypi:`flake8-docstrings`.  There are no user-visible changes.
```

Comment:
nit: This *is* a user visible change isn't it? Docstrings show up in the documentation!

Reply:
Most of the diff is just adding or deleting empty lines, and while I also reworded or fixed the formatting of a few docstrings, almost none of them are public API or included in the docs!  On closer inspection the docstring for `assume()` now begins "Calling `assume` ..." instead of "`assume()` ...", but I'll round that to 'no visible impact' 😉

Basically, we started from a high base - this pull is mostly about in-code layout and future-proofing.

Reply:
Answering my own nit: Yes but none of the public facing functions got changed as part of this.

Reply:
Oops. I failed to refresh and thus missed that you'd already answered.

Do we actually expose the `assume` docstring anywhere at the moment? Anyway, either way I think this is fine as is.

Reply:
https://hypothesis.readthedocs.io/en/latest/details.html#hypothesis.assume